### PR TITLE
Use actual options description for readme

### PR DESCRIPTION
**Pull request by RusinovAnton**
@beck I've found that README has outdated options description

### CODE REVIEW

**Comment by stefanor** [comment]
I think we should try to avoid reformatting and reordering everything, so it is easier to keep track of our delta from upstream.

**Comment by beck** [comment]
I think the two options added [here](https://github.com/yola/jsxgettext/pull/3/files#diff-f8849ae9f0614ba9464fdd14c2d98dc0R23) could be added to the readme and any future conflict will be easy to resolve.

Going forward, I would love to push the json changes upstream but that is blocked by two necessary refactors:

1. Upstream needs to decouple the language parses from the core library, more about that discussion here: https://github.com/zaach/jsxgettext/issues/82#issuecomment-81922341

1. The core json parser needs to be decoupled from the gulp plugin, more about that here:
https://github.com/yola/gulp-static-i18n/issues/16#issuecomment-88211310

Despite being awfully clunky, everything generally "works"—which kills the motivation to spend time here when there are other, more important, priorities.

**Comment by RusinovAnton** [comment]
I just wanted it was in sync with actual help output:
image

**Comment by stefanor** [comment]
Right, and I think the best thing to do there is to start with a PR to the upstream to do this. And then our single line addition in our fork.

File: README.md [hunk 1]
```diff
@@ -41,14 +41,17 @@ Or from source:
   
     Options:
   
-      -h, --help                      output usage information
```
Comment (beck) [inline README.md]: This change needs to made up-stream (not in our fork)

File: README.md [hunk 2]
```diff
@@ -41,14 +41,17 @@ Or from source:
...
+    -p, --output-dir <path>         output files will be placed in directory <path>
+    -k, --keyword [keywords]        additional keywords to be looked for
+    -j, --join-existing             join messages with existing file
+    -L, --language [lang]           use the specified language (javascript, ejs, jinja, handlebars, jade, json, swig) [javascript]
+    -s, --sanity                    sanity check during the extraction
+    --project-id-version [version]  This is the project name and version of the generated package/catalog.
+    --report-bugs-to [bug address]  An email address or URL where you can report bugs in the untranslated strings.
+    -c, --add-comments [tag]        place comment blocks starting with TAG and preceding keyword lines in output file (default: "L10n:").
+    --debug                         Print each filename being prased.
+    --keys [keys]                   JSON keys (comma separated) to use when extracting messages.
```
Comment (beck) [inline README.md]: This line (and the line below) are the only changes I'd expect to see in this pull.

**Comment by RusinovAnton** [comment]
Updated. I feel like it would be better if I close this request and create another one with prettier commit, wouldn't it?

File: README.md [hunk 3]
```diff
@@ -51,4 +51,6 @@ Or from source:
       -s, --sanity                    sanity check during the extraction
       --project-id-version [version]  This is the project name and version of the generated package/catalog.
       --report-bugs-to [bug address]  An email address or URL where you can report bugs in the untranslated strings.
-      -c, --add-comments [tag]        place comment blocks starting with TAG and preceding keyword lines in output file (default: "L10n:").
\ No newline at end of file
+      -c, --add-comments [tag]        place comment blocks starting with TAG and preceding keyword lines in output file (default: "L10n:").
```
Comment (stefanor) [inline README.md:54]: We can leave that trailing whitespace untouched.

File: README.md [hunk 4]
```diff
@@ -53,4 +53,4 @@ Or from source:
       --report-bugs-to [bug address]  An email address or URL where you can report bugs in the untranslated strings.
       -c, --add-comments [tag]        place comment blocks starting with TAG and preceding keyword lines in output file (default: "L10n:").
       --keys [keys]                   JSON keys (comma separated) to use when extracting messages.
-      --ignore-keys [keys]            JSON keys to exclude when extracting messages.
+      --ignore-keys [keys]            JSON keys to exclude when extracting messages.
```
Comment (stefanor) [inline README.md]: I don't think this commit changed the line you meant to change.

Comment (RusinovAnton) [reply README.md]: I couldn't see difference between master and my branch except blank line at the end.

Comment (stefanor) [reply README.md]: The difference was on the line I commented on. This line is a line you added, so I don't know why you'd want to have trailing whitespace on it.

**Pull request closed without merging**